src/libs/OptionsListUtils.js

```diff
@@ -404,20 +466,24 @@ function getOptions(reports, personalDetails, activeReportID, {
 
     const allReportOptions = [];
     _.each(orderedReports, (report) => {
+        if (!report) {
+            return null;
```

Comment:
```suggestion
            return;
```

Should work too or was there a reason to return `null` to move to the next iteration?

Reply:
Ah, no. There was no reason for the `null`.